spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+describe "Contestants index page:" do
+  before(:each) do
+    # Challenges =================================================================
+    @recycled_material_challenge = Challenge.create!(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create!(theme: "Apartment Furnishings", project_budget: 1000)
+    # Projects =================================================================
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+    # Contestants =================================================================
+    @jay = Contestant.create!(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create!(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create!(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    @erin = Contestant.create!(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+    ContestantProject.create!(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create!(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    ContestantProject.create!(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    ContestantProject.create!(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    ContestantProject.create!(contestant_id: @kentaro.id, project_id: @boardfit.id)
+    ContestantProject.create!(contestant_id: @erin.id, project_id: @boardfit.id)
+  end
+
+  describe "When I visit the contestants index page ('/contestants')" do
+    it "I see a list of names of all the contestants And under each contestants name I see a list of the projects (names) that they've been on" do
+      visit "/contestants"
+
+      expect(page).to have_content("#{@jay.name}")
+      expect(page).to have_content("#{@gretchen.name}")
+      expect(page).to have_content("#{@kentaro.name}")
+      expect(page).to have_content("#{@erin.name}")
+      
+      expect(page).to have_content("#{@jay.projects.name}")
```

Comment:
You could make this test a bit more robust by using `within` blocks (and adding classes or ids to the elements view) so that you can make sure that Gretchen's projects are listed under Gretchen and *not* under Jay or Kentaro.